ci/appveyor/package.cfg

```diff
@@ -8,7 +8,7 @@ display_name = aeternity for Windows
 
 description = The full aeternity node for Windows.
 
-version = 1.5.0-alpha.1
+version = 2.0.0-rc.1
```

Comment:
@tolbrino As mentioned, please notice removal of `-alpha.1`.
@tolbrino Please advise if this line can be removed altogether, or at least set as a placeholder value not needing to be updated at each release (and not giving impression of being intended to be kept in line with package version).

Reply:
It needs to be updated, because it is used by the installer.

Reply:
[Mentioned in release process](https://github.com/aeternity/aeternity/wiki/Release-process/_compare/7d243f049c27e3898d91ce5c312f76b3a359c6e4...d9dae97b1cbf33ea9ac7628cda8421997c736cfe).
@tolbrino Thanks.